platforms/xilinx_alveo_u250/cl_firesim/design/bitstream_config.xdc

```diff
@@ -0,0 +1,12 @@
+# The majority of these constraints are redudant as them come with the u250 given XDC
+set_property CONFIG_VOLTAGE 1.8                        [current_design]
+set_property CONFIG_MODE {SPIx4} 											 [current_design]
+set_property BITSTREAM.CONFIG.CONFIGFALLBACK Enable    [current_design]                  ;# Golden image is the fall back image if  new bitstream is corrupted.
```

Comment:
Nit: Can you fix the spacing in this file (spacing before the `;` and the `[`)

Reply:
done